### PR TITLE
Bugfix/small fixes nino

### PR DESCRIPTION
**Pull request by NinoRy**
#

### CODE REVIEW

File: modules/ProvBase/Http/Controllers/ContractController.php [hunk 1]
```diff
@@ -208,7 +208,7 @@ public function prepare_rules($rules, $data)
     {
         if (\Module::collections()->has('PropertyManagement')) {
             // Only group contracts without modems can belong to a Realty directly - with CMs realty_id must be null
-            if ($data['realty_id'] && isset($data['id'])) {
+            if (isset($data['realty_id']) && $data['realty_id'] && isset($data['id'])) {
```
Comment (cschra) [inline modules/ProvBase/Http/Controllers/ContractController.php]: Why is this needed anyways? $data['realty_id'] is not used here

Comment (NinoRy) [reply modules/ProvBase/Http/Controllers/ContractController.php]: There is a sense behind, but it's not worth to explain it, as it will be removed by the propertymanagement branch anyway
